src/test/java/org/mapstruct/intellij/inspection/WrongUsageOfMappersFactoryInspectionTest.java

```diff
@@ -51,6 +51,8 @@ public void testWrongUsageOfMappersFactory() {
         myFixture.launchAction( allQuickFixes.get( 6 ) ); // Remove jsr330 componentModel
         myFixture.launchAction( allQuickFixes.get( 8 ) ); // Remove custom componentModel
 
+        assertThat( myFixture.getAllQuickFixes() ).isEmpty();
```

Comment:
Why is this needed?

Reply:
I forgot to mention it, sorry, could've left a comment. This is a workaround for the latest EAP Intellij. The test assertion just fails and only there, and only this test. I have no Idea why.

During inspecting it inside the debugger, and after evaluating `myFixture.getAllQuickFixes()` the assertion suddenly worked. So this is a workaround for that flaky test (or flaky fixture/assertion) - I will leave a comment with a TODO there, I don't think you have the time desire and to investigate this? 😅

Reply:
>I will leave a comment with a TODO there, I don't think you have the time desire and to investigate this? 

I am a bit busy yes, but I would prefer to figure out why is it happening, since it might be happening in the actual EAP as well. Can you please extract this change into a different PR so I can investigate it?

Reply:
The error is only happening in the LATEST-EAP-SNAPSHOT. See: https://github.com/mapstruct/mapstruct-idea/actions/runs/6129983187
And this is also happening in the master branch.
So you can just reproduce this by changing the ideaVersion (see git patch, no separate PR needed 🙂 )

I applied this workaround to get the build green. I can of course revert the workaround and we can merge the PR although the build is failing (when all issues are resolved ofc).

```
Subject: [PATCH] change current build to LATEST-EAP-SNAPSHOT
---
Index: gradle.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/gradle.properties b/gradle.properties
--- a/gradle.properties	(revision 5168fbab259c968771f2944a98eb0187b55f5ade)
+++ b/gradle.properties	(date 1694897260461)
@@ -2,10 +2,10 @@
 # https://www.jetbrains.com/intellij-repository/releases
 # https://www.jetbrains.com/intellij-repository/snapshots
 
-ideaVersion = 2021.2
+ideaVersion = LATEST-EAP-SNAPSHOT
 ideaType = IC
 sources = true
-isEAP = false
+isEAP = true
 runGenerators = true
 
 pluginGroup = org.mapstruct

```